### DISCUSSION TITLE
Flatpak (www.flathub.org) version is outdated

### DISCUSSION THREAD

**Discussion by 38github** [jamulussoftware/jamulus#2966] [Technical Support & Help]
The Flatpak (www.flathub.org) version is outdated and the current provided one is:

```
Updated
August 12, 2022

Version
3_9_0nightly1
```

**Comment by hoffie** [comment]
Hrm, to be honest, I don't think FlatHub is part of any official release process. I think we once built Flatpaks as part of Github CI. Flathub submission seems to have been done by @joao-p-marques in the past (#803).

@joao-p-marques Is that still true? Could you update it? Should we maybe ping you explicitly when releasing new versions?

**Reply by joao-p-marques** [reply]
Hi! Sorry for the late response. Yes, it is true. 

The flathub version is a bit old because of the error that https://github.com/flathub/io.jamulus.Jamulus/pull/174 tries to fix. 

However, it seems that it does need some updates... Help wanted 😃
